test/ctim/schemas/campaign_test.cljc

```diff
@@ -59,8 +59,8 @@
                                   :source "source"
                                   :relationship "rel"}]
           :confidence "High"
-          :activity {:date_time #inst "2016-02-11T00:40:48.212-00:00"
-                     :description "activity"}})))
+          :activity [{:date_time #inst "2016-02-11T00:40:48.212-00:00"
```

Comment:
I've fixed this in #106 and it is unrelated to the issue.
So it should be cleaner to simply remove these modification along this issue. Furthermore #106 as an updated doc.